src/provider/qti.js

```diff
@@ -251,16 +251,7 @@ var qtiProvider = {
 
             //catch server errors
             var submitError = function submitError(err) {
-                if (err && err.unrecoverable){
-                    self.trigger(
-                        'alert.error',
-                        __(
-                            'An unrecoverable error occurred. Your test session will be paused.'
-                        )
-                    );
-
-                    self.trigger('pause', {message : err.message});
-                } else if (err.code === 200) {
```

Comment:
**nitpick:** I don't think this is the proper way to address the issue. This will revert fixes added by https://github.com/oat-sa/tao-test-runner-qti-fe/pull/189 and https://github.com/oat-sa/tao-test-runner-qti-fe/pull/54

**suggestion:** instead of killing the symptom, I'd rather suggest to investigate at the root cause, which may be at the place where the `unrecoverable` attribute is added to the error. A solution, could be to better qualify the error, and distinguish the testState issue from the others. The attribute is set here: https://github.com/oat-sa/tao-test-runner-qti-fe/blob/d27e84ccf41fae87870ac67c514338cbe99d045b/src/proxy/cache/proxy.js#L217